bin/common.bash

```diff
@@ -351,6 +351,8 @@ validate_config() {
             fi
         done
 
+        yajsv -s "${config_template_path}/schemas/config.yaml" "${merged_config}"
```

Comment:
Could you substitute the file prefix and replace it with `sc|wc-config.yaml`?

And catch if it finds errors and set `maybe_exit` to true?

Reply:
- [ ] Could you substitute the file prefix and replace it with `sc|wc-config.yaml`?
- [x] And catch if it finds errors and set `maybe_exit` to true?